src/Nowruz/modules/general/components/avatarDropDown/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { Divider, Typography } from '@mui/material';
```

Comment:
I suggest 2 atomic components (Avatar and dropdown) and this would be a general container